examples/flutter_gallery/lib/demo/cupertino/cupertino_picker_demo.dart

```diff
@@ -70,20 +71,23 @@ class _CupertinoPickerDemoState extends State<CupertinoPickerDemo> {
         child: new GestureDetector(
           // Blocks taps from propagating to the modal sheet and popping.
           onTap: () {},
-          child: new CupertinoPicker(
-            scrollController: scrollController,
-            itemExtent: _kPickerItemHeight,
-            backgroundColor: CupertinoColors.white,
-            onSelectedItemChanged: (int index) {
-              setState(() {
-                _selectedItemIndex = index;
-              });
-            },
-            children: new List<Widget>.generate(coolColorNames.length, (int index) {
-              return new Center(child:
-                new Text(coolColorNames[index]),
-              );
-            }),
+          child: new SafeArea(
+            bottom: true,
```

Comment:
This is true by default - is the intent to have top/left/right be false?

Reply:
ah thanks, removed. I actually just want it safe everywhere.